docker/rhel/Dockerfile

```diff
@@ -6,5 +6,5 @@ WORKDIR /app
 
 RUN dnf install -y redhat-rpm-config gcc libffi-devel python38-devel openssl-devel
 RUN pip3 install --upgrade pip
-RUN pip3 install -r requirements.txt
+#RUN pip3 install -r requirements.txt
```

Comment:
Looks like the build failed because the dev dependencies need to be specified separately given `pip install .` only installs required ones

Reply:
I put all the dev dependencies in the `tests_requires` list in `setup.py`, we'll see if that works.

Reply:
I'm trying the former. If that doesn't work, we'll try the latter.